tasklist/client/src/modules/internationalization/locales/de.json

```diff
@@ -149,7 +149,7 @@
     "customFiltersModalConfirmDeletionButton": "Löschen bestätigen",
     "customFiltersModalAriaLabel": "Benutzerdefinierte Filtermodale",
     "customFiltersModalApplyFiltersTitle": "Filter anwenden",
-    "customFiltersModalCloseButton": "Schließen",
+    "optionsModalCloseButton": "Schließen",
```

Comment:
Should we instead simply split into two keys?

Reply:
@Skaiir I renamed it because the close action is used in multiple dialogs (and can have the same tooltip).
Alternatively, we can split into separate keys.

Reply:
Probably not a huge deal.